tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -72,9 +74,7 @@ func (r *Textual) GetFieldValueRenderer(fd protoreflect.FieldDescriptor) (ValueR
 		fd.Kind() == protoreflect.Uint64Kind ||
 		fd.Kind() == protoreflect.Int32Kind ||
 		fd.Kind() == protoreflect.Int64Kind:
-		{
-			return NewIntValueRenderer(), nil
-		}
+		return NewIntValueRenderer(), nil
 
 	case fd.Kind() == protoreflect.StringKind:
```

Comment:
This condition is duplicated: https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=BUG&id=cosmos_cosmos-sdk&open=AYTE5cUYJ1y5ta4n9bJ5

Reply:
Thanks! I'll fix it in my PR then: #13600